dask_ml/model_selection/_incremental.py

```diff
@@ -471,6 +503,16 @@ def _check_array(self, X, **kwargs):
         X = check_array(X, **kwargs)
         return X
 
+    def _handle_verbosity(self, verbose):
```

Comment:
I'm not sure that a `.fit` call should affect this. Ideally all the configuration would be done when the module is loaded. You might want to look at how distributed handles logging. I think by default we should follow the config setting there, and users can override it if needed.